jaxrs/src/test/java/se/fortnox/reactivewizard/ExceptionHandlerTest.java

```diff
@@ -152,6 +153,22 @@ void shouldLogCancellationExceptionAtDebugLevel() {
         }
     }
 
+    @Test
+    void shouldLogNativeIoExceptionAtDebugLevel() {
```

Comment:
could you add test with a ioexception thrown with 

a. null message
b. other message

Reply:
Fixed